### ISSUE TITLE
Having a true plain text editor

### ISSUE THREAD

**Issue by saccarosium** [nextcloud/notes#1010]
**Is your feature request related to a problem? Please describe.**
I don't use markdown I usually do asciidoc and custom formated plaintext. When I open this file the editor impones markdown symtax and tries to preview it.

**Describe the solution you'd like**
Loading custom exention and .txt in a "plain" editor with no formatting and preview but only text.

**Describe alternatives you've considered**
I could do with other app would be great having a multiplatform option.

**Comment by brainchild0** [comment]
The application should select an appropriate editor based on file extension. Selection for Markdown editing should occur for files ending in `.md`. 

For files ending in `.txt`, a simple  text box should be presented suitable for plain text, styled in a system-width font.

Any further features for a text editor may be helpful, but at minimum, text files should not be interpreted as Markdown, nor displayed in a variable-width typesetting.

**Comment by joshtrichards** [comment]
Aren't you both just requesting the txt file functionality that exists in the Nextcloud Android client already?

**Comment by jackduckworth** [comment]
> Aren't you both just requesting the txt file functionality that exists in the Nextcloud Android client already?

I don't think so - the Android client allows you to open a txt file, but if there is anything in the text resembling markup it immediately is formatted. And, annoyingly, there is no way to turn this off. 

I know many people love markup, but there are some of us that just want to work with plain text. We are finding that apps, that used to work perfectly, are now just butchering our notes with random formatting. 

We **could** re-format out notes to remove/escape markup but:
1. if you ever paste source code, or, say, a bash script, into a text editor you will often find it re-formatted to be unreadable
2. I don't want to do that

I feel this app already does everything I need perfectly, but the newer functionality is breaking it for my use case. And now with every markup enhancement, it is becoming less usable for plain text.

If anyone can work out a good point in the code to turn off markup, I'd be happy to build a forked branch.

**Comment by jackduckworth** [comment]
I've realised that I'm complaining in the wrong forum. 

I never actually use the browser to edit notes, and cant see myself ever wanting to.

So I've created an issue in the Android Notes app github: https://github.com/nextcloud/notes-android/issues/1930
